src/components/Card.jsx

```diff
@@ -1,26 +1,52 @@
+import { useState } from 'react';
+
 import { calculateDateDifference, formatDate } from '../utils/dateCalculate';
+
 import defaultImg from '../assets/logo.svg';
 import starIcon from '../assets/icon/star.svg';
 import kebabIcon from '../assets/icon/kebab.svg';
 
 import './Card.css'
+import ModalLinkAdd from './modals/ModalLinkAdd';
+
+const Card = ({imgSrc, createdAt, title, description, url, folderName, folderList}) => {
+  const [isOpenKebab, setIsOpenKebab] = useState(false);
+  const [isOpenModalLinkAdd, setIsOpenModalLinkAdd] = useState(false);
+
+  const handleClickKebab = () => {
+    setIsOpenKebab(!isOpenKebab);
```

Comment:
상태를 변경할 때 이전 상태를 이용할때는 updater function을 사용해주는게 좋아요~
[참조](https://react.dev/learn/queueing-a-series-of-state-updates)
```suggestion
    setIsOpenKebab((isOpenKebab) => !isOpenKebab);
```